Sources/Controllers/MessagesViewController.swift

```diff
@@ -251,49 +251,77 @@ extension MessagesViewController: UICollectionViewDataSource {
         return messagesLayoutDelegate.footerViewSize(for: message, at: indexPath, in: messagesCollectionView)
     }
 
+    open override func viewWillTransition(to size: CGSize, with coordinator: UIViewControllerTransitionCoordinator) {
```

Comment:
iMessage nor Facebook messenger support remembering the scroll position on rotation. So it may be better to leave this behavior to the user to decide if it is what they want

Reply:
Well, at least iMessage remembers that is was pointing to the bottom before the rotation. I think this a polite behavior towards the user. Maybe this - keeping scrolling position at bottom after rotation - should be addressed in another PR. What do you think?

Reply:
I agree this should be addressed in another PR. :)